tools/build_api.py

```diff
@@ -424,7 +424,8 @@ def get_mbed_official_release(version, profile=None):
             tuple(
                 [
                     TARGET_MAP[target].name,
-                    tuple(['ARM', 'GCC_ARM'])
+                    tuple(transform_release_toolchains(
```

Comment:
may I suggest please change 
https://github.com/ARMmbed/mbed-os/blob/master/tools/build_api.py#L388
and
https://github.com/ARMmbed/mbed-os/blob/master/tools/build_api.py#L398
to `if version in [ '5' , '6' ]`
otherwise if passing '6' this API will not work

Reply:
do we pass '6' ? It's not defined anywhere in the tools currently

Reply:
I updated the PR

Reply:
This is what I am not sure, it is not defined, but it is a valid use case

Reply:
I don't understand why we are introducing "6"

Reply:
+1, I would not introduce 6 in the tools . Let's stay with what we are . If we introduce 6, it should be with updated tools (not the current form). As adding 6, we are creating new flow (in reality, its the same as 5 here so not needed).

I would revert that 6 addition

Reply:
I got it ready locally, can push :)

Reply:
fixed